task/buildah-oci-ta/0.2/buildah-oci-ta.yaml

```diff
@@ -475,96 +475,24 @@ spec:
           memory: 256Mi
       securityContext:
         runAsUser: 0
-    - name: merge-syft-sboms
-      image: registry.access.redhat.com/ubi9/python-39:1-192.1722518946@sha256:0176b477075984d5a502253f951d2502f0763c551275f9585ac515b9f241d73d
+    - name: prepare-sboms
+      image: quay.io/redhat-appstudio/sbom-utility-scripts-image@sha256:53a3041dff341b7fd1765b9cc2c324625d19e804b2eaff10a6e6d9dcdbde3a91
       workingDir: /var/workdir
       script: |
-        #!/bin/python3
-        import json
+        echo "Merging contents of sbom-source.json and sbom-image.json into sbom-cyclonedx.json"
+        python3 /scripts/merge_syft_sboms.py
```

Comment:
As the next improvement, we should make these scripts accept params for the sbom paths, similar to the merge_cachi2_sboms.py script.

In the current state, the scripts that don't take params just look like magic :D

Reply:
I agree, let's do that as a next improvement (not part of this round of PRs)